README.md

```diff
@@ -1,16 +1,27 @@
-# datadog-mobile-app-run-tests
+# Datadog Continuous Testing for Bitrise
 
-placeholder description
+<!-- TODO add link to marketplace after we publish the step -->
+<!-- [![Visual Studio Marketplace Version]()][A-1]  -->
+<!-- [![Build Status](https://dev.azure.com/datadog-ci/Datadog%20CI%20Azure%20DevOps%20Extension/_apis/build/status%2FDevelopment?branchName=main)](https://dev.azure.com/datadog-ci/Datadog%20CI%20Azure%20DevOps%20Extension/_build/latest?definitionId=4&branchName=main) -->
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# Overview
 
-## How to use this Step
+With the `synthetics-test-automation-bitrise-step-run-tests` step you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][B-2].
 
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
+## Setup
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+To get started:
+
+1. Add this step to your workflow. You can see Bitrise's documentation on how to do that [here][D-1]. You can also configure it locally by referencing this step in your `bitrise.yml`
+2. Add you API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found [here][D-2]
+3. Configure your step inputs ([docs][D-3]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs will be described later on.
+
+## How to use this Step locally
+
+Can be run directly with the [bitrise CLI][B-3], just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line and call `bitrise run test`.
```

Comment:
Looks like `git clone` and `cd` are already described in the step-by-step process below.